developer/docker-compose.yml

```diff
@@ -3,8 +3,6 @@ services:
     container_name: postgres
     image: postgres:15.2
```

Comment:
As long as we're updating this file, can you update this to 15.7 ? (I think that's the latest minor release in Postgres 15).

Reply:
sure